_posts/2024-04-01-review-topic-sentiment-overview.md

```diff
@@ -0,0 +1,56 @@
+
+*“Sehr schönes Hotel mit freundlichen Personal, sehr Hilfsbereit und sehr freundlich und sehr Sauber schöner Pool. Sehr schön Zimmer.”* (translation: *Very nice hotel with friendly staff, very helpful and very friendly and very clean, beautiful pool. Very nice room.*) 
+
+When it comes to our choice for which language models we use, there are a number of criteria at stake that need to be considered. Our choice in using GPT 3.5 was determined by a trade-off between quality of results (esp. with German language extraction) and budget. At the same time, we're exploring different avenues to use (fine-tuned) open source models as a replacement. Our experience in this respect has mostly shown two challenges: First, we still lack the same quality in the results for sentiment and topic extraction with the German language when using OSS models. Second, we also experience difficulties in achieving the same level of reliability for generating structured output in JSON format. Nevertheless, we are very confident that we will soon overcome these obstacles and use self-hosted models for these purposes. 
+
+These challenges underscore the complexities of leveraging advanced AI technologies in a way that respects the authenticity and diversity of user experiences. 
+
+## Evaluation 
+
+We invite you to read our tech blog post ([https://techblog.holidaycheck.com/post/2023/08/24/sentiment-analysis-llms](https://techblog.holidaycheck.com/post/2023/08/24/sentiment-analysis-llms)) where we outline how we test our sentiment analysis approaches. Within our teams, manual checks are a cornerstone of our evaluation process, providing a hands-on approach to refining and enhancing our models. We are committed to continuous improvement, relying on these meticulous reviews to polish our algorithms. Moreover, the feature's introduction to our users follows a strategic, gradual roll-out. This phased approach allows us to monitor the quality closely, make adjustments, and ensure that the insights we provide are both accurate and representative of the original review. 
+
+## Conclusion 
+
+By leveraging LLMs to extract and analyze topics and sentiments from user reviews, HolidayCheck is not only enhancing the travel planning experience but also fostering a deeper connection within our community. This is a testament to our commitment to leveraging cutting-edge technology to truly understand and meet the needs of travelers worldwide. 
```

Comment:
By leveraging LLMs to extract and analyze topics and sentiments from user reviews, HolidayCheck is enhancing the travel planning experience and fostering a deeper connection within our community. This is a testament to our commitment to leveraging cutting-edge technology to truly understand and meet the needs of travellers worldwide.